src/services/embedService.ts

```diff
@@ -3,16 +3,17 @@ import { COLOR } from '../../utilities/types/globals';
 import { Category } from '../database/entities/category.entity';
 import { ReactRole } from '../database/entities/reactRole.entity';
 import RoleBot from '../../src/bot';
-import { GET_REACT_ROLES_BY_CATEGORY_ID } from '../database/database';
 import * as tutorialJson from '../../utilities/json/tutorial.json';
 import { Colors } from '../interfaces';
 import { codeBlock } from '@discordjs/builders';
 import { AVATAR_URL } from '../vars';
+import { GET_REACT_ROLES_BY_CATEGORY_ID } from '../database/queries/reactRole.query';
 
 export class EmbedService {
   private static userTagInfo = (user: User | string): string => {
```

Comment:
Did you know functions which take varying typed arguments in the same runtime end up causing de-optimization in the V8 runtime? The suggested alternative is to have different implementations, e.g.
```ts
  private static userTagInfo = (user: User): string => this.userTagInfoString(user.tag, user.id);
  private static userTagInfoString = (userTag: string, userId: string = userTag): string => {
    return `${userTag} (<@${userId}>)`;
  };
```